### PR TITLE
✨ feat: add distrobox docs

### PR DESCRIPTION
**Pull request by dude-with-the-long-username**

### CODE REVIEW

File: distrobox [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+---
+
+
+# Create a distrobox using the Ubuntu Linux image:
+
+distrobox-create <container_name></container_name> --image <ubuntu:latest>
+
+# List all distrobox containers with verbose information:
+
+distrobox-list -v
```
Comment (EmilySeville7cfg) [inline distrobox:12]:
```suggestion
distrobox-list --verbose
```

Optional change: long options are much easy to remember.

File: distrobox [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+# Execute command on the host, while inside of a container:
+
+distrobox-host-exec <command>
+
+# Export an app (atom) from the container to the host (will show up in your host system's application list):
+
+distrobox-export --app <atom> --extra-flags "--foreground"
+
+# Export a binary(ranger) from the container to the host:
```
Comment (EmilySeville7cfg) [inline distrobox:26]:
```suggestion
# Export a binary (ranger) from the container to the host:
```

Optional change: missing space.

**Review by EmilySeville7cfg — APPROVED** [review]
- Approved because no mandatory changes found.
- This page should be reviewed by other `distrobox` users.
- Thanks for a contribution. ❤️

**Comment by chrisallenlane** [comment]
Merged with some minor whitespace edits. Thanks!

**Pull request merged**